releasing.md

```diff
@@ -23,19 +23,24 @@ Run the E2E test workflow using our release cluster
 [kubeflow/testing#42](https://github.com/kubeflow/testing/issues/42) will simplify this.
 
 ```
-PULL_BASE_SHA=<commit to build>
-DATE=`date +%Y%m%d`
-VERSION_TAG="v${DATE}-${PULL_BASE_SHA}"
 JOB_NAME="tf-operator-release"
-JOB_TYPE=postsubmit
+JOB_TYPE=tf-operator-release
```

Comment:
So is this also a predefined type like presubmit, postsubmit? The type value matches the job name so just trying to understand.

Reply:
The job type is defined by prow.
https://github.com/kubernetes/test-infra/tree/master/prow

The reason we set job type here is because the scripts invoked by our E2E tests e.g.
https://github.com/kubeflow/testing/blob/master/images/checkout.sh

Might be using JOB_TYPE. For example, checkout.sh might be using JOB_TYPE to determine whether its a post submit or presubmit job which affects how the script checks out the code (e.g. do we add refs to pull the PR.)

Our checkout.sh script doesn't use JOB_TYPE right now so technically we'd be fine not setting it. But we do anyway because the contract that checkout.sh is built around is that JOB_TYPE will be set.